src/widgetastic_patternfly5/components/menus/select.py

```diff
@@ -19,7 +20,11 @@ class BaseSelect:
     https://www.patternfly.org/components/menus/select
     """
 
-    BUTTON_LOCATOR = ".//button"
+    BUTTON_LOCATOR = (
```

Comment:
Better to overwrite in widget itself. Those limitation can break usage.  Not tested against console dot so.